src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -585,7 +586,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 				await clientSession.kernel.ready;
 				await this.updateKernelInfoOnKernelChange(clientSession.kernel, kernelAlias);
 			}
-			if (clientSession.isInErrorState) {
+			if (clientSession.isInErrorState && clientSession.errorMessage) {
```

Comment:
Is there any situation where we could miss an actual error here because it didn't have a message?

Reply:
Hm.. I can removed the errorMessage check in case the message is an empty string